code/__HELPERS/materials.dm

```diff
@@ -0,0 +1,5 @@
+/// Gets the reference for the material type that was given
+#define getmaterialref(A) (SSmaterials.materials[A])
+
+/// Checks if the material given is actually a material, or a category.
+#define ismaterialcategory(A) (!istype(A, /datum/material) && !ispath(A, /datum/material) )
```

Comment:
Well the thing is, it's very unclear otherwise what the check actually does. I mostly need it when going through a list of materials for when you want to make a design, not sure how else to do it.

Reply:
I don't see how this is unclear if done normally without the macro. You can do istext() if you prefer?

Reply:
That works,

I figured that people would not understand what the concept of categories is, but I suppose they shouldnt be fucking with the code if they dont want to go find out themselves.